<div class="bibliography"><style type="text/css" id="zoterostylesheet" scoped>
.bibshowhide {display:none;}
.abstract {display:none;}
.blink {margin:0;margin-right:15px;padding:0;display:none;}
</style>
<script type="text/javascript">
 function downloadFile(elem) {
  filename = "article.ris"
  if (elem.parentNode) {
    var elems = elem.parentNode.getElementsByTagName('*');
    for (i in elems) {
        if((' ' + elems[i].className + ' ').indexOf(' ' + 'bibshowhide' + ' ') > -1) 
           {
  var ee = elems[i]
  if (ee.childNodes[0]) { ee = ee.childNodes[0] } 
  var pom = document.createElement('a');
  pom.setAttribute('href', 'data:text/plain;charset=utf-8,' + encodeURIComponent(ee.innerHTML));
  pom.setAttribute('download', filename);
  pom.click();
}}}}
function show(elem) {
  if (elem.parentNode) {
   var elems = elem.parentNode.getElementsByTagName('*'), i;
    for (i in elems) {
        if((' ' + elems[i].className + ' ').indexOf(' ' + 'bibshowhide' + ' ') > -1) 
           { if (elems[i].style.display == 'block') {elems[i].style.display = 'none';} else {elems[i].style.display = 'block';}}}}
  return(void(0));}
function changeCSS() {
	if (!document.styleSheets) return;
	var theRules = new Array();
    //ss = document.styleSheets[document.styleSheets.length-1];
    var ss = document.getElementById('zoterostylesheet');
    if (ss) {
    ss = ss.sheet
	if (ss.cssRules)
		theRules = ss.cssRules
	else if (ss.rules)
		theRules = ss.rules
	else return;
	theRules[theRules.length-2].style.display = 'inline';
    theRules[theRules.length-1].style.display = 'inline';
    }
    }
changeCSS();</script><hr>
<div class="full-bib-section"><div class="bib-item"><div class="csl-entry">Beerenwinkel, N., Greenman, C. D., &amp; Lagergren, J. (2016). <a class="doctitle" href="http://dx.doi.org/10.1371/journal.pcbi.1004717">Computational Cancer Biology: An Evolutionary Perspective.</a> <i>PLoS Comput Biol</i>, <i>12</i>(2), e1004717. http://doi.org/10.1371/journal.pcbi.1004717</div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">bib</a><div class="bibshowhide"><div class="bib">@article{beerenwinkel_computational_2016,
	title = {Computational {Cancer} {Biology}: {An} {Evolutionary} {Perspective}},
	volume = {12},
	shorttitle = {Computational {Cancer} {Biology}},
	url = {http://dx.doi.org/10.1371/journal.pcbi.1004717},
	doi = {10.1371/journal.pcbi.1004717},
	number = {2},
	urldate = {2016-02-05TZ},
	journal = {PLoS Comput Biol},
	author = {Beerenwinkel, Niko and Greenman, Chris D. and Lagergren, Jens},
	month = feb,
	year = {2016},
	keywords = {cancer, concepts, darwinian, epigenetics, evolution, mathematical modeling, review, statistics, variant},
	pages = {e1004717}
}</div></div></div><div class="blink"><a title="Download EndNote record" href="javascript:downloadFile(this);" onclick="downloadFile(this);">ris</a><div class="bibshowhide"><div class="ris">TY  - JOUR
TI  - Computational Cancer Biology: An Evolutionary Perspective
AU  - Beerenwinkel, Niko
AU  - Greenman, Chris D.
AU  - Lagergren, Jens
T2  - PLoS Comput Biol
DA  - 2016/02/04/
PY  - 2016
DO  - 10.1371/journal.pcbi.1004717
DP  - PLoS Journals
VL  - 12
IS  - 2
SP  - e1004717
J2  - PLoS Comput Biol
ST  - Computational Cancer Biology
UR  - http://dx.doi.org/10.1371/journal.pcbi.1004717
Y2  - 2016/02/05/T20:21:50Z
KW  - cancer
KW  - concepts
KW  - darwinian
KW  - epigenetics
KW  - evolution
KW  - mathematical modeling
KW  - review
KW  - statistics
KW  - variant
ER  -</div></div></div></div><div class="bib-item"><div class="csl-entry">Kaufman, C. K., Mosimann, C., Fan, Z. P., Yang, S., Thomas, A. J., Ablain, J., … Zon, L. I. (2016). <a class="doctitle" href="http://science.sciencemag.org/content/351/6272/aad2197">A zebrafish melanoma model reveals emergence of neural crest identity during melanoma initiation.</a> <i>Science</i>, <i>351</i>(6272), aad2197. http://doi.org/10.1126/science.aad2197</div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">abstract</a><div class="bibshowhide"><div class="abstract">Visualizing the beginnings of melanoma
In cancer biology‚ a tumor begins from a single cell within a group of precancerous cells that share genetic mutations. Kaufman et al. used a zebrafish melanoma model to visualize cancer initiation (see the Perspective by Boumahdi and Blanpain). They used a fluorescent reporter that specifically lit up neural crest progenitors that are only present during embryogenesis or during adult melanoma tumor formation. The appearance of this tumor correlated with a set of gene regulatory elements‚ called super-enhancers‚ whose identification and manipulation may prove beneficial in detecting and preventing melanoma initiation.
Science‚ this issue p. 10.1126/science.aad3867; see also p. 453
Structured Abstract
INTRODUCTIONThe “cancerized field” concept posits that cells in a given tissue sharing an oncogenic mutation are cancer-prone‚ yet only discreet clones within the field initiate tumors. Studying the process of cancer initiation has remained challenging because of (i) the rarity of these events‚ (ii) the difficulty of visiualizing initiating clones in living organisms‚ and (iii) the transient nature of a newly transformed clone emerging before it expands to form an early tumor. A more complete understanding of the molecular processes that regulate cancer initiation could provide important prognostic information about which precancerous lesions are most prone to becoming cancer and also implicate druggable molecular pathways that‚ when inhibited‚ may prevent the cancer from ever starting.
RATIONALEThe majority of benign nevi carry oncogenic BRAFV600E mutations and can be considered a cancerized field of melanocytes‚ but they only rarely convert to melanoma. In an effort to define events that initiate cancer‚ we used a melanoma model in the zebrafish in which the human BRAFV600E oncogene is driven by the melanocyte-specific mitfa promoter. When bred into a p53 mutant background‚ these fish develop melanoma tumors over the course of many months. The zebrafish crestin gene is expressed embryonically in neural crest progenitors (NCPs) and is specifically reexpressed only in melanoma tumors‚ making it an ideal candidate for tracking melanoma from initiation onward.
RESULTSWe developed a crestin:EGFP reporter that recapitulates the embryonic neural crest expression pattern of crestin and its expression in melanoma tumors. We show through live imaging of transgenic zebrafish crestin reporters that within a cancerized field (BRAFV600E-mutant; p53-deficient)‚ a single melanocyte reactivates the NCP state‚ and this establishes that a fate change occurs at melanoma initiation in this model. Early crestin+ patches of cells expand and are transplantable in a manner consistent with their possessing tumorigenic activity‚ and they exhibit a gene expression pattern consistent with the NCP identity readout by the crestin reporter. The crestin element is regulated by NCP transcription factors‚ including sox10. Forced sox10 overexpression in melanocytes accelerated melanoma formation‚ whereas CRISPR/Cas9 targeting of sox10 delayed melanoma onset. We show activation of super-enhancers at NCP genes in both zebrafish and human melanomas‚ identifying an epigenetic mechanism for control of this NCP signature leading to melanoma.
CONCLUSIONThis work using our zebrafish melanoma model and in vivo reporter of NCP identity allows us to see cancer from its birth as a single cell and shows the importance of NCP-state reemergence as a key event in melanoma initiation from a field of cancer-prone melanocytes. Thus‚ in addition to the typical fixed genetic alterations in oncogenes and tumor supressors that are required for cancer development‚ the reemergence of progenitor identity may be an additional rate-limiting step in the formation of melanoma. Preventing NCP reemergence in a field of cancer-prone melanocytes may thus prove therapeutically useful‚ and the association of NCP genes with super-enhancer regulatory elements implicates the associated druggable epigenetic machinery in this process. Download high-res image Open in new tab Download Powerpoint Neural crest reporter expression in melanoma.The crestin:EGFP transgene is specifically expressed in melanoma in BRAFV600E/p53 mutant melanoma-prone zebrafish. (Top) A single cell expressing crestin:EGFP expands into a small patch of cells over the course of 2 weeks‚ capturing the initiation of melanoma formation (bracket). (Bottom) A fully formed melanoma specifically expresses crestin:EGFP‚ whereas the rest of the fish remains EGFP-negative.
The “cancerized field” concept posits that cancer-prone cells in a given tissue share an oncogenic mutation‚ but only discreet clones within the field initiate tumors. Most benign nevi carry oncogenic BRAFV600E mutations but rarely become melanoma. The zebrafish crestin gene is expressed embryonically in neural crest progenitors (NCPs) and specifically reexpressed in melanoma. Live imaging of transgenic zebrafish crestin reporters shows that within a cancerized field (BRAFV600E-mutant; p53-deficient)‚ a single melanocyte reactivates the NCP state‚ revealing a fate change at melanoma initiation in this model. NCP transcription factors‚ including sox10‚ regulate crestin expression. Forced sox10 overexpression in melanocytes accelerated melanoma formation‚ which is consistent with activation of NCP genes and super-enhancers leading to melanoma. Our work highlights NCP state reemergence as a key event in melanoma initiation.
Melanocytes with oncogenic or tumor suppressor mutations revert to expressing the crestin gene early in melanoma formation. [Also see Perspective by Boumahdi and Blanpain]
Melanocytes with oncogenic or tumor suppressor mutations revert to expressing the crestin gene early in melanoma formation. [Also see Perspective by Boumahdi and Blanpain]</div></div></div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">bib</a><div class="bibshowhide"><div class="bib">@article{kaufman_zebrafish_2016,
	title = {A zebrafish melanoma model reveals emergence of neural crest identity during melanoma initiation},
	volume = {351},
	copyright = {Copyright © 2016, American Association for the Advancement of Science},
	issn = {0036-8075, 1095-9203},
	url = {http://science.sciencemag.org/content/351/6272/aad2197},
	doi = {10.1126/science.aad2197},
	language = {en},
	number = {6272},
	urldate = {2016-01-29TZ},
	journal = {Science},
	author = {Kaufman, Charles K. and Mosimann, Christian and Fan, Zi Peng and Yang, Song and Thomas, Andrew J. and Ablain, Julien and Tan, Justin L. and Fogley, Rachel D. and Rooijen, Ellen van and Hagedorn, Elliott J. and Ciarlo, Christie and White, Richard M. and Matos, Dominick A. and Puller, Ann-Christin and Santoriello, Cristina and Liao, Eric C. and Young, Richard A. and Zon, Leonard I.},
	month = jan,
	year = {2016},
	keywords = {Development, SKCM, classics, epigenetics, evolution, melanoma, super\_enhancer, zebrafish},
	pages = {aad2197}
}</div></div></div><div class="blink"><a title="Download EndNote record" href="javascript:downloadFile(this);" onclick="downloadFile(this);">ris</a><div class="bibshowhide"><div class="ris">TY  - JOUR
TI  - A zebrafish melanoma model reveals emergence of neural crest identity during melanoma initiation
AU  - Kaufman, Charles K.
AU  - Mosimann, Christian
AU  - Fan, Zi Peng
AU  - Yang, Song
AU  - Thomas, Andrew J.
AU  - Ablain, Julien
AU  - Tan, Justin L.
AU  - Fogley, Rachel D.
AU  - Rooijen, Ellen van
AU  - Hagedorn, Elliott J.
AU  - Ciarlo, Christie
AU  - White, Richard M.
AU  - Matos, Dominick A.
AU  - Puller, Ann-Christin
AU  - Santoriello, Cristina
AU  - Liao, Eric C.
AU  - Young, Richard A.
AU  - Zon, Leonard I.
T2  - Science
AB  - Visualizing the beginnings of melanoma
In cancer biology, a tumor begins from a single cell within a group of precancerous cells that share genetic mutations. Kaufman et al. used a zebrafish melanoma model to visualize cancer initiation (see the Perspective by Boumahdi and Blanpain). They used a fluorescent reporter that specifically lit up neural crest progenitors that are only present during embryogenesis or during adult melanoma tumor formation. The appearance of this tumor correlated with a set of gene regulatory elements, called super-enhancers, whose identification and manipulation may prove beneficial in detecting and preventing melanoma initiation.
Science, this issue p. 10.1126/science.aad3867; see also p. 453
Structured Abstract
INTRODUCTIONThe “cancerized field” concept posits that cells in a given tissue sharing an oncogenic mutation are cancer-prone, yet only discreet clones within the field initiate tumors. Studying the process of cancer initiation has remained challenging because of (i) the rarity of these events, (ii) the difficulty of visiualizing initiating clones in living organisms, and (iii) the transient nature of a newly transformed clone emerging before it expands to form an early tumor. A more complete understanding of the molecular processes that regulate cancer initiation could provide important prognostic information about which precancerous lesions are most prone to becoming cancer and also implicate druggable molecular pathways that, when inhibited, may prevent the cancer from ever starting.
RATIONALEThe majority of benign nevi carry oncogenic BRAFV600E mutations and can be considered a cancerized field of melanocytes, but they only rarely convert to melanoma. In an effort to define events that initiate cancer, we used a melanoma model in the zebrafish in which the human BRAFV600E oncogene is driven by the melanocyte-specific mitfa promoter. When bred into a p53 mutant background, these fish develop melanoma tumors over the course of many months. The zebrafish crestin gene is expressed embryonically in neural crest progenitors (NCPs) and is specifically reexpressed only in melanoma tumors, making it an ideal candidate for tracking melanoma from initiation onward.
RESULTSWe developed a crestin:EGFP reporter that recapitulates the embryonic neural crest expression pattern of crestin and its expression in melanoma tumors. We show through live imaging of transgenic zebrafish crestin reporters that within a cancerized field (BRAFV600E-mutant; p53-deficient), a single melanocyte reactivates the NCP state, and this establishes that a fate change occurs at melanoma initiation in this model. Early crestin+ patches of cells expand and are transplantable in a manner consistent with their possessing tumorigenic activity, and they exhibit a gene expression pattern consistent with the NCP identity readout by the crestin reporter. The crestin element is regulated by NCP transcription factors, including sox10. Forced sox10 overexpression in melanocytes accelerated melanoma formation, whereas CRISPR/Cas9 targeting of sox10 delayed melanoma onset. We show activation of super-enhancers at NCP genes in both zebrafish and human melanomas, identifying an epigenetic mechanism for control of this NCP signature leading to melanoma.
CONCLUSIONThis work using our zebrafish melanoma model and in vivo reporter of NCP identity allows us to see cancer from its birth as a single cell and shows the importance of NCP-state reemergence as a key event in melanoma initiation from a field of cancer-prone melanocytes. Thus, in addition to the typical fixed genetic alterations in oncogenes and tumor supressors that are required for cancer development, the reemergence of progenitor identity may be an additional rate-limiting step in the formation of melanoma. Preventing NCP reemergence in a field of cancer-prone melanocytes may thus prove therapeutically useful, and the association of NCP genes with super-enhancer regulatory elements implicates the associated druggable epigenetic machinery in this process. Download high-res image Open in new tab Download Powerpoint Neural crest reporter expression in melanoma.The crestin:EGFP transgene is specifically expressed in melanoma in BRAFV600E/p53 mutant melanoma-prone zebrafish. (Top) A single cell expressing crestin:EGFP expands into a small patch of cells over the course of 2 weeks, capturing the initiation of melanoma formation (bracket). (Bottom) A fully formed melanoma specifically expresses crestin:EGFP, whereas the rest of the fish remains EGFP-negative.
The “cancerized field” concept posits that cancer-prone cells in a given tissue share an oncogenic mutation, but only discreet clones within the field initiate tumors. Most benign nevi carry oncogenic BRAFV600E mutations but rarely become melanoma. The zebrafish crestin gene is expressed embryonically in neural crest progenitors (NCPs) and specifically reexpressed in melanoma. Live imaging of transgenic zebrafish crestin reporters shows that within a cancerized field (BRAFV600E-mutant; p53-deficient), a single melanocyte reactivates the NCP state, revealing a fate change at melanoma initiation in this model. NCP transcription factors, including sox10, regulate crestin expression. Forced sox10 overexpression in melanocytes accelerated melanoma formation, which is consistent with activation of NCP genes and super-enhancers leading to melanoma. Our work highlights NCP state reemergence as a key event in melanoma initiation.
Melanocytes with oncogenic or tumor suppressor mutations revert to expressing the crestin gene early in melanoma formation. [Also see Perspective by Boumahdi and Blanpain]
Melanocytes with oncogenic or tumor suppressor mutations revert to expressing the crestin gene early in melanoma formation. [Also see Perspective by Boumahdi and Blanpain]
DA  - 2016/01/29/
PY  - 2016
DO  - 10.1126/science.aad2197
DP  - science.sciencemag.org
VL  - 351
IS  - 6272
SP  - aad2197
LA  - en
SN  - 0036-8075, 1095-9203
UR  - http://science.sciencemag.org/content/351/6272/aad2197
Y2  - 2016/01/29/T14:40:26Z
KW  - Development
KW  - SKCM
KW  - classics
KW  - epigenetics
KW  - evolution
KW  - melanoma
KW  - super_enhancer
KW  - zebrafish
ER  -</div></div></div></div><div class="bib-item"><div class="csl-entry">Lipinski, K. A., Barber, L. J., Davies, M. N., Ashenden, M., Sottoriva, A., &amp; Gerlinger, M. (2016). <a class="doctitle" href="http://www.sciencedirect.com/science/article/pii/S2405803315000692">Cancer Evolution and the Limits of Predictability in Precision Cancer Medicine.</a> <i>Trends in Cancer</i>, <i>2</i>(1), 49–63. http://doi.org/10.1016/j.trecan.2015.11.003</div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">abstract</a><div class="bibshowhide"><div class="abstract">The ability to predict the future behavior of an individual cancer is crucial for precision cancer medicine. The discovery of extensive intratumor heterogeneity and ongoing clonal adaptation in human tumors substantiated the notion of cancer as an evolutionary process. Random events are inherent in evolution and tumor spatial structures hinder the efficacy of selection‚ which is the only deterministic evolutionary force. This review outlines how the interaction of these stochastic and deterministic processes‚ which have been extensively studied in evolutionary biology‚ limits cancer predictability and develops evolutionary strategies to improve predictions. Understanding and advancing the cancer predictability horizon is crucial to improve precision medicine outcomes.</div></div></div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">bib</a><div class="bibshowhide"><div class="bib">@article{lipinski_cancer_2016,
	title = {Cancer {Evolution} and the {Limits} of {Predictability} in {Precision} {Cancer} {Medicine}},
	volume = {2},
	issn = {2405-8033},
	url = {http://www.sciencedirect.com/science/article/pii/S2405803315000692},
	doi = {10.1016/j.trecan.2015.11.003},
	number = {1},
	urldate = {2016-01-30TZ},
	journal = {Trends in Cancer},
	author = {Lipinski, Kamil A. and Barber, Louise J. and Davies, Matthew N. and Ashenden, Matthew and Sottoriva, Andrea and Gerlinger, Marco},
	month = jan,
	year = {2016},
	keywords = {classics, evolution, heterogeneity, resistance, review},
	pages = {49--63}
}</div></div></div><div class="blink"><a title="Download EndNote record" href="javascript:downloadFile(this);" onclick="downloadFile(this);">ris</a><div class="bibshowhide"><div class="ris">TY  - JOUR
TI  - Cancer Evolution and the Limits of Predictability in Precision Cancer Medicine
AU  - Lipinski, Kamil A.
AU  - Barber, Louise J.
AU  - Davies, Matthew N.
AU  - Ashenden, Matthew
AU  - Sottoriva, Andrea
AU  - Gerlinger, Marco
T2  - Trends in Cancer
AB  - The ability to predict the future behavior of an individual cancer is crucial for precision cancer medicine. The discovery of extensive intratumor heterogeneity and ongoing clonal adaptation in human tumors substantiated the notion of cancer as an evolutionary process. Random events are inherent in evolution and tumor spatial structures hinder the efficacy of selection, which is the only deterministic evolutionary force. This review outlines how the interaction of these stochastic and deterministic processes, which have been extensively studied in evolutionary biology, limits cancer predictability and develops evolutionary strategies to improve predictions. Understanding and advancing the cancer predictability horizon is crucial to improve precision medicine outcomes.
DA  - 2016/01//
PY  - 2016
DO  - 10.1016/j.trecan.2015.11.003
DP  - ScienceDirect
VL  - 2
IS  - 1
SP  - 49
EP  - 63
J2  - Trends in Cancer
SN  - 2405-8033
UR  - http://www.sciencedirect.com/science/article/pii/S2405803315000692
Y2  - 2016/01/30/T22:58:32Z
KW  - classics
KW  - evolution
KW  - heterogeneity
KW  - resistance
KW  - review
ER  -</div></div></div></div><div class="bib-item"><div class="csl-entry">Trapnell, C. (2015). <a class="doctitle" href="http://genome.cshlp.org/content/25/10/1491">Defining cell types and states with single-cell genomics.</a> <i>Genome Research</i>, <i>25</i>(10), 1491–1498. http://doi.org/10.1101/gr.190595.115</div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">abstract</a><div class="bibshowhide"><div class="abstract">A revolution in cellular measurement technology is under way: For the first time‚ we have the ability to monitor global gene regulation in thousands of individual cells in a single experiment. Such experiments will allow us to discover new cell types and states and trace their developmental origins. They overcome fundamental limitations inherent in measurements of bulk cell population that have frustrated efforts to resolve cellular states. Single-cell genomics and proteomics enable not only precise characterization of cell state‚ but also provide a stunningly high-resolution view of transitions between states. These measurements may finally make explicit the metaphor that C.H. Waddington posed nearly 60 years ago to explain cellular plasticity: Cells are residents of a vast “landscape” of possible states‚ over which they travel during development and in disease. Single-cell technology helps not only locate cells on this landscape‚ but illuminates the molecular mechanisms that shape the landscape itself. However‚ single-cell genomics is a field in its infancy‚ with many experimental and computational advances needed to fully realize its full potential.</div></div></div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">bib</a><div class="bibshowhide"><div class="bib">@article{trapnell_defining_2015,
	title = {Defining cell types and states with single-cell genomics},
	volume = {25},
	issn = {1088-9051, 1549-5469},
	url = {http://genome.cshlp.org/content/25/10/1491},
	doi = {10.1101/gr.190595.115},
	language = {en},
	number = {10},
	urldate = {2015-10-14TZ},
	journal = {Genome Research},
	author = {Trapnell, Cole},
	month = oct,
	year = {2015},
	pmid = {26430159},
	keywords = {Waddington, capacitance, chromatin, classics, epigenetics, evolution, genomics, ideas, oped, review, single-cell},
	pages = {1491--1498}
}</div></div></div><div class="blink"><a title="Download EndNote record" href="javascript:downloadFile(this);" onclick="downloadFile(this);">ris</a><div class="bibshowhide"><div class="ris">TY  - JOUR
TI  - Defining cell types and states with single-cell genomics
AU  - Trapnell, Cole
T2  - Genome Research
AB  - A revolution in cellular measurement technology is under way: For the first time, we have the ability to monitor global gene regulation in thousands of individual cells in a single experiment. Such experiments will allow us to discover new cell types and states and trace their developmental origins. They overcome fundamental limitations inherent in measurements of bulk cell population that have frustrated efforts to resolve cellular states. Single-cell genomics and proteomics enable not only precise characterization of cell state, but also provide a stunningly high-resolution view of transitions between states. These measurements may finally make explicit the metaphor that C.H. Waddington posed nearly 60 years ago to explain cellular plasticity: Cells are residents of a vast “landscape” of possible states, over which they travel during development and in disease. Single-cell technology helps not only locate cells on this landscape, but illuminates the molecular mechanisms that shape the landscape itself. However, single-cell genomics is a field in its infancy, with many experimental and computational advances needed to fully realize its full potential.
DA  - 2015/10/01/
PY  - 2015
DO  - 10.1101/gr.190595.115
DP  - genome.cshlp.org
VL  - 25
IS  - 10
SP  - 1491
EP  - 1498
J2  - Genome Res.
LA  - en
SN  - 1088-9051, 1549-5469
UR  - http://genome.cshlp.org/content/25/10/1491
Y2  - 2015/10/14/T06:07:23Z
KW  - Waddington
KW  - capacitance
KW  - chromatin
KW  - classics
KW  - epigenetics
KW  - evolution
KW  - genomics
KW  - ideas
KW  - oped
KW  - review
KW  - single-cell
ER  -</div></div></div></div><div class="bib-item"><div class="csl-entry">Jacobs, F. M. J., Greenberg, D., Nguyen, N., Haeussler, M., Ewing, A. D., Katzman, S., … Haussler, D. (2014). <a class="doctitle" href="http://www.nature.com/nature/journal/v516/n7530/full/nature13760.html">An evolutionary arms race between KRAB zinc-finger genes ZNF91/93 and SVA/L1 retrotransposons.</a> <i>Nature</i>, <i>516</i>(7530), 242–245. http://doi.org/10.1038/nature13760</div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">abstract</a><div class="bibshowhide"><div class="abstract">Throughout evolution primate genomes have been modified by waves of retrotransposon insertions. For each wave‚ the host eventually finds a way to repress retrotransposon transcription and prevent further insertions. In mouse embryonic stem cells‚ transcriptional silencing of retrotransposons requires KAP1 (also known as TRIM28) and its repressive complex‚ which can be recruited to target sites by KRAB zinc-finger (KZNF) proteins such as murine-specific ZFP809 which binds to integrated murine leukaemia virus DNA elements and recruits KAP1 to repress them. KZNF genes are one of the fastest growing gene families in primates and this expansion is hypothesized to enable primates to respond to newly emerged retrotransposons. However‚ the identity of KZNF genes battling retrotransposons currently active in the human genome‚ such as SINE-VNTR-Alu (SVA) and long interspersed nuclear element 1 (L1)‚ is unknown. Here we show that two primate-specific KZNF genes rapidly evolved to repress these two distinct retrotransposon families shortly after they began to spread in our ancestral genome. ZNF91 underwent a series of structural changes 8-12 million years ago that enabled it to repress SVA elements. ZNF93 evolved earlier to repress the primate L1 lineage until [sim]12.5 million years ago when the L1PA3-subfamily of retrotransposons escaped ZNF93/’s restriction through the removal of the ZNF93-binding site. Our data support a model where KZNF gene expansion limits the activity of newly emerged retrotransposon classes‚ and this is followed by mutations in these retrotransposons to evade repression‚ a cycle of events that could explain the rapid expansion of lineage-specific KZNF genes.</div></div></div><div class="blink"><a href="javascript:show(this);" onclick="show(this);">bib</a><div class="bibshowhide"><div class="bib">@article{jacobs_evolutionary_2014,
	title = {An evolutionary arms race between {KRAB} zinc-finger genes {ZNF}91/93 and {SVA}/{L}1 retrotransposons},
	volume = {516},
	copyright = {© 2014 Nature Publishing Group, a division of Macmillan Publishers Limited. All Rights Reserved.},
	issn = {0028-0836},
	url = {http://www.nature.com/nature/journal/v516/n7530/full/nature13760.html},
	doi = {10.1038/nature13760},
	language = {en},
	number = {7530},
	urldate = {2015-10-12TZ},
	journal = {Nature},
	author = {Jacobs, Frank M. J. and Greenberg, David and Nguyen, Ngan and Haeussler, Maximilian and Ewing, Adam D. and Katzman, Sol and Paten, Benedict and Salama, Sofie R. and Haussler, David},
	month = dec,
	year = {2014},
	keywords = {classics, genereg, ideas, interactions, lncrna, network, transposon},
	pages = {242--245}
}</div></div></div><div class="blink"><a title="Download EndNote record" href="javascript:downloadFile(this);" onclick="downloadFile(this);">ris</a><div class="bibshowhide"><div class="ris">TY  - JOUR
TI  - An evolutionary arms race between KRAB zinc-finger genes ZNF91/93 and SVA/L1 retrotransposons
AU  - Jacobs, Frank M. J.
AU  - Greenberg, David
AU  - Nguyen, Ngan
AU  - Haeussler, Maximilian
AU  - Ewing, Adam D.
AU  - Katzman, Sol
AU  - Paten, Benedict
AU  - Salama, Sofie R.
AU  - Haussler, David
T2  - Nature
AB  - Throughout evolution primate genomes have been modified by waves of retrotransposon insertions. For each wave, the host eventually finds a way to repress retrotransposon transcription and prevent further insertions. In mouse embryonic stem cells, transcriptional silencing of retrotransposons requires KAP1 (also known as TRIM28) and its repressive complex, which can be recruited to target sites by KRAB zinc-finger (KZNF) proteins such as murine-specific ZFP809 which binds to integrated murine leukaemia virus DNA elements and recruits KAP1 to repress them. KZNF genes are one of the fastest growing gene families in primates and this expansion is hypothesized to enable primates to respond to newly emerged retrotransposons. However, the identity of KZNF genes battling retrotransposons currently active in the human genome, such as SINE-VNTR-Alu (SVA) and long interspersed nuclear element 1 (L1), is unknown. Here we show that two primate-specific KZNF genes rapidly evolved to repress these two distinct retrotransposon families shortly after they began to spread in our ancestral genome. ZNF91 underwent a series of structural changes 8-12 million years ago that enabled it to repress SVA elements. ZNF93 evolved earlier to repress the primate L1 lineage until [sim]12.5 million years ago when the L1PA3-subfamily of retrotransposons escaped ZNF93/'s restriction through the removal of the ZNF93-binding site. Our data support a model where KZNF gene expansion limits the activity of newly emerged retrotransposon classes, and this is followed by mutations in these retrotransposons to evade repression, a cycle of events that could explain the rapid expansion of lineage-specific KZNF genes.
DA  - 2014/12/11/
PY  - 2014
DO  - 10.1038/nature13760
DP  - www.nature.com
VL  - 516
IS  - 7530
SP  - 242
EP  - 245
J2  - Nature
LA  - en
SN  - 0028-0836
UR  - http://www.nature.com/nature/journal/v516/n7530/full/nature13760.html
Y2  - 2015/10/12/T21:07:51Z
KW  - classics
KW  - genereg
KW  - ideas
KW  - interactions
KW  - lncrna
KW  - network
KW  - transposon
ER  -</div></div></div></div></div></div>
